frontend/packages/portal-frontend/src/compound/doseCurvesTab/DoseCurvesMainContent.tsx

```diff
@@ -109,24 +96,16 @@ function DoseCurvesMainContent({
           );
         },
       },
-      {
-        accessor: "modelId",
-        Header: "Model ID",
-        maxWidth: 120,
-        minWidth: 80,
-      },
-      {
-        accessor: "auc",
-        Header: "AUC",
-        maxWidth: 120,
-        minWidth: 80,
-      },
```

Comment:
It's nice that you were able to move some of these configs to the `utils.ts` file, especially now that there are more columns 👍